docs/synchronizations-faq.md

```diff
@@ -0,0 +1,43 @@
+# Synchronizations FAQ
+##
+
+In Private Packagist synchronizations is how you keep packages, teams, their members, and access permissions in sync with your version control system like GitHub, GitLab or Bitbucket.
+
+You can find below answers for frequently-asked questions concerning synchronizations.
+
+## What happens if you promote a synchronization to a primary synchronization?
+
+Private Packagist organizations that are synchronized with a remote organization have automatically all admins and owners of the synchronization assigned to the admins and owners team in Private Packagist.
+
+If an organization is synchronized with multiple remote organizations then only admins and owners of the primary synchronization are added to the respective teams.
```

Comment:
Do you think instead of `added` we can use a different verb that also implies that admins and owners that were in the old GitHub org but not in the new org will be removed now that the paragraph below only talk about switching integration?

Reply:
how about `matches` or `synchronized` instead of `added` ?

Reply:
Would probably avoid `synchronized` because that was is already heavily used in this article

Reply:
ok, used matched here, anything else you think is still unclear?